src/topics.js

```diff
@@ -1,369 +1,373 @@
-"use strict";

```

Comment:
My editor made some whitespace changes here for some reason, I'll figure it out (unless you guys know why this is happening, I'm developing in IntelliJ on OSX)